test/interactions/interactionTests.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Interactions", () => {
+  describe("Interaction", () => {
```

Comment:
The tests interface is extremely thin if you look at the url.  It literally does a grep for the string you pressed and runs those according tests